apis/python/tests/test_registration_mappings.py

```diff
@@ -434,6 +441,8 @@ def test_multiples_without_experiment(
             measurement_name="measname",
             ingest_mode="write",
         )
+        shutil.rmtree("/tmp/soma", ignore_errors=True)
```

Comment:
Delete?

Reply:
Thanks @ryan-williams 
https://github.com/single-cell-data/TileDB-SOMA/pull/2281/commits/afd6d7b5658452921c3d0e7927620632d144ef10
-- this was from #2242, same debug cruft as there